<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rrPlayer</class>
 <widget class="QMainWindow" name="rrPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>1280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="isPlay">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>90</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>播放</string>
     </property>
    </widget>
    <widget class="Slider" name="progressSlider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>800</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
</string>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Slider" name="volumeSlider">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>20</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string>音量调节</string>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
     <property name="value">
      <number>128</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
   </widget>
   <widget class="VideoWidget" name="video">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="open">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="openFile"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>播放</string>
    </property>
    <addaction name="play"/>
    <addaction name="pause"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视频</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>音频</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>字幕</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>查看</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_8">
    <property name="title">
     <string>选项</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_9">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="open"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menu_7"/>
   <addaction name="menu_8"/>
   <addaction name="menu_9"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openFile">
   <property name="text">
    <string>文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="pause">
   <property name="text">
    <string>暂停</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="play">
   <property name="text">
    <string>播放</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Slider</class>
   <extends>QSlider</extends>
   <header>slider.h</header>
  </customwidget>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>openFile</sender>
   <signal>triggered()</signal>
   <receiver>rrPlayer</receiver>
   <slot>OpenFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isPlay</sender>
   <signal>clicked()</signal>
   <receiver>rrPlayer</receiver>
   <slot>PlayOrPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>675</y>
    </hint>
    <hint type="destinationlabel">
     <x>-25</x>
     <y>678</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>progressSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>rrPlayer</receiver>
   <slot>SliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>664</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>764</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>progressSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>rrPlayer</receiver>
   <slot>SliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>736</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>rrPlayer</receiver>
   <slot>volumeSliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1225</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>1229</x>
     <y>758</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>rrPlayer</receiver>
   <slot>volumeSliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1186</x>
     <y>671</y>
    </hint>
    <hint type="destinationlabel">
     <x>1189</x>
     <y>748</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenFile()</slot>
  <slot>PlayOrPause()</slot>
  <slot>SliderPress()</slot>
  <slot>SliderRelease()</slot>
  <slot>volumeSliderRelease()</slot>
  <slot>volumeSliderPress()</slot>
 </slots>
</ui>
